Vizzuality/forest-atlas-landscape-cms
# Remove Function and Tags and add owner, edited and created to dataset…

Add new information to datasets table.

## Testing instructions

1. Access to the datasets table of any site
2. Check that function and tags columns don't appear
3. Check that owner, created and edited columns appear

## Pivotal Tracker

[https://www.pivotaltracker.com/story/show/170235518](https://www.pivotaltracker.com/story/show/170235518)
